js/commerce_bluesnap.form.js

```diff
@@ -0,0 +1,252 @@
+/**
+ * @file
+ * Javascript to submit card details to BlueSnap in a PCI-compliant way.
+ */
```

Comment:
**NOT A CHANGE REQUEST**

Drupal Core has switched to using ECMAScript 6 for its javascript files; create an issue on Drupal.org for doing the same at some point later on.